### DISCUSSION TITLE
How are you using PenPot?

### DISCUSSION THREAD

**Discussion by Quin452** [uxbox/uxbox#632] [General]
Just a question for the community really.

I've moved from Adobe XD, and used it for websites, a few apps, and even some game assets.
Now, I've chatted with the PP (hehe) team, and they use the app for social media posts too.

So I'm curious, what are you guys using it for?
What practises are you using?

I love the Artboard feature (XD has something similar), and generally use them as pages, but PenPot also has Pages, which I've not understood the reason for.

Which is another reason why I post this discussion 😄

**Comment by myfunnyandy** [comment]
It's been only a week since the alpha release so it might be a bit soon for most of you to answer this question, but we already have been designing Penpot with Penpot for a while. Some of our current uses are:
- Wireframing
- Visual design
- Interaction design (connecting artboards to simulate navigation)
- Whiteboard collaborating sessions (we have a component for postits)
- Marketing pieces (newsletters, social media images, slideshows)
- Ilustration (simple ones for the moment)
- Memes (a lot)

You can find some examples on twitter under the hashtag #madewithpenpot, it would be nice more people share their work that way. Don't be shy ;)

**Comment by myfunnyandy** [comment]
About the pages, we use them to separate different stages of the design process at the same document.

For example, those are the pages of the file where we are working on the design of the user's documentation:
![Screenshot from 2021-02-11 16-04-54](https://user-images.githubusercontent.com/708713/107656588-9cf95180-6c84-11eb-8675-b4ecdd025a73.png)

There are many other reasons to work with pages, It would be really interesting if someone else share theirs.

**Reply by Quin452** [reply]
That makes really good sense!  I'd call them something else (what, I don't know) to avoid the confusion I had earlier 😛
